Paco/assets/custom_base.js

```diff
@@ -187,13 +187,23 @@ var paco = (function (init) {
       };
 
       function getLastEvent() {
-        return JSON.parse(window.db.getLastEvent());
+        return JSON.parse(window.db.getLastNEvents(num));
```

Comment:
Yes. Good catch. I was playing around with the function name and did not test it. It needs to be changed to '1'.